scripts/OGC_tests/install_TE.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "Installing TE Source"
+mkdir repo
+cd repo
+git clone https://github.com/opengeospatial/teamengine.git
```

Comment:
You can skip lines 4 and 5 by doing
`git clone https://github.com/opengeospatial/teamengine.git repo`